tester/src/test/scala/spinal/tester/scalatest/SpinalSimVerilatorIoTest.scala

```diff
@@ -201,10 +202,12 @@ class SpinalSimVerilatorIoTest extends AnyFunSuite {
           newEnumTest(dut.nativeEncoding)
           newEnumTest(dut.binaryOneHotEncoding)
           newEnumTest(dut.binarySequentialEncoding)
+          newEnumTest(dut.graySequentialEncoding)
         }
       }
     }
 
+    println(prefix)
```

Comment:
Please be careful on the print-out statements.

Reply:
sure, was still debugging some stuff there, gonna get removed